httpbp/example_server_test.go

```diff
@@ -107,8 +107,9 @@ func ExampleNewBaseplateServer() {
 		Endpoints:   handlers.Endpoints(),
 		Middlewares: []httpbp.Middleware{loggingMiddleware},
 	})
+	args := baseplate.ServeArgs{Server: server}
```

Comment:
It's a bit weird to define `args` before the `err` check. In general I'd prefer to just inline it in the line it's used, especially in examples. Same for the thrift example.